ramls/hold.json

```diff
@@ -41,10 +41,6 @@
       "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$",
       "description": "The UUID of the pick up location"
     },
-    "queueLength": {
-      "type": "integer",
-      "description": "The number of patrons with holds on this item"
-    },
     "queuePosition": {
```

Comment:
does the order of the element in the JSON document matter? I see queuePosition in various places in many  JSON samples in this PR, sometimes before Item, other times after.

Reply:
No. Position doesn't matter for JSON schema, at least I know of no keywords in JSON schema that require ordering of fields. If there were such syntax it would certainly have to be specified in some way. The schemas we write have no ordering requirements, apart from the values in JSON arrays.

Reply:
Ah, I skipped the description. I need some sleep.....